Add files via upload

mazaquax opened the pull request.

mazaquax committed:
Add files via upload

illarionovEA reviewed (changes requested):
в задаче 4 не просили массив из 10 единиц, внимательно посмотри условие задачи
нет задачи 5
kmeans должен быть в виде функции и работать в данными любой размерности и любым числом кластеров

mazaquax committed:
Add files via upload

Исправил 4,5 задачи и сделал многомерный k-means.
https://www.codewars.com/users/mazaquax

mazaquax commented:
Исправил 4,5 задачи и сделал многомерный k-means.
https://www.codewars.com/users/mazaquax
